SemanticMediaWiki.php

```diff
@@ -117,13 +117,13 @@ class_alias( 'SMW\ParameterProcessorFactory', 'SMW\ParameterFormatterFactory' );
 define( 'SEMANTIC_EXTENSION_TYPE', true );
 
 // Load global constants
-require_once( __DIR__ . '/includes/Defines.php' );
+require_once ( __DIR__ . '/includes/Defines.php' );
 
 // Temporary measure to ease Composer/MW 1.22 migration
 require_once __DIR__ . '/includes/NamespaceManager.php';
 
 // Load global functions
-require_once( __DIR__ . '/includes/GlobalFunctions.php' );
+require_once ( __DIR__ . '/includes/GlobalFunctions.php' );
```

Comment:
This seems odd. The old style is better IMO. Even better is the following, which probably also makes the CS thing shut up:

```php
require_once __DIR__ . '/includes/GlobalFunctions.php'
```